includes/view/User_view.php

```diff
@@ -412,14 +413,18 @@ function User_view_myshifts(
         foreach ($myshifts_table as $i => &$shift) {
             $before = $myshifts_table[$i - 1] ?? null;
             $after = $myshifts_table[$i + 1] ?? null;
-            if (Carbon::now() > $shift['start'] &&  Carbon::now() < $shift['end']) {
-                $shift['row-class'] = 'border border-info border-2';
-            } elseif ($after && Carbon::now() > $shift['end'] && Carbon::now() < $after['start']) {
-                $shift['row-class'] = 'border-bottom border-info';
-            } elseif (!$before && Carbon::now() < $shift['start']) {
-                $shift['row-class'] = 'border-top-info';
-            } elseif (!$after && Carbon::now() > $shift['end']) {
-                $shift['row-class'] = 'border-bottom border-info';
+            if ($shift['freeloaded']) {
+                $shift['row-class'] = 'border border-danger border-2';
+            } else {
```

Comment:
@xuwhite that could be just another elseif here?

Reply:
nope that is an else on purpose

Reply:
But what?